troi/patches/daily_jams.py

```diff
@@ -111,7 +111,10 @@ def create(self, inputs, patch_args):
         artist_filter = troi.filters.ArtistCreditFilterElement([])
         artist_filter.set_sources(r_lookup)
 
-        artist_limiter = troi.filters.ArtistCreditLimiterElement()
+        if type == "top":
```

Comment:
This is an attempt do make the top artists list better... Will likely change again.